src/main/java/dev/pgm/community/translations/TranslationConfig.java

```diff
@@ -0,0 +1,41 @@
+package dev.pgm.community.translations;
+
+import dev.pgm.community.feature.config.FeatureConfigImpl;
+import java.util.List;
+import org.bukkit.configuration.Configuration;
+
+public class TranslationConfig extends FeatureConfigImpl {
+
+  private static final String KEY = "translations";
```

Comment:
Let's add an API Key field, as I'm going to add it very soon: `translations.api-key`

Reply:
If provided, it should add an `Authorization: Bearer {api-key}` header to each request.

Reply:
Great idea! Added 👍